### ISSUE TITLE
Security key authentication causes runtime error.

### ISSUE THREAD

**Issue by yourjelly** [authelia/authelia#2984]
### Description

I tried adding a security key (just a usb-c yubikey) as my two factor authentication and it threw a red error in the corner saying `There was an issue retrieving global configuration`

And then the Authelia container died with this in the logs.
```
panic: runtime error: invalid memory address or nil pointer dereference

[signal SIGSEGV: segmentation violation code=0x1 addr=0x200 pc=0x562d72b0d7a7]

goroutine 7329 [running]:
github.com/valyala/fasthttp.(*RequestCtx).Done(0x55f0061a7568)
github.com/valyala/fasthttp@v1.33.0/server.go:2691 +0x7
context.propagateCancel.func1()
context/context.go:280 +0x50
created by context.propagateCancel
```
After restarting the container I tried logging in using the security key and was able to successfully login (the cookie was created) but the Authelia died again and needed to be restarted.
```
panic: runtime error: invalid memory address or nil pointer dereference

[signal SIGSEGV: segmentation violation code=0x1 addr=0x200 pc=0x56505321a7a7]

goroutine 275 [running]:
github.com/valyala/fasthttp.(*RequestCtx).Done(0x565052dca568)
github.com/valyala/fasthttp@v1.33.0/server.go:2691 +0x7
context.propagateCancel.func1()
context/context.go:280 +0x50
created by context.propagateCancel
context/context.go:278 +0x1d0
```

Switching back to one-time codes for 2fa continues to work with no issue.
Before registering using my physical security key, was able to successfully register my pixel4 as a security key using chrome without this runtime error.

**Comment by james-d-elliott** [comment]
What version are you running? Also try logging out and in.

**Comment by yourjelly** [comment]
```
time="2022-03-09T23:29:40-05:00" level=info msg="Authelia v4.34.3 is starting"
time="2022-03-09T23:29:40-05:00" level=info msg="Log severity set to info"
time="2022-03-09T23:29:40-05:00" level=info msg="Storage schema is being checked for updates"
time="2022-03-09T23:29:40-05:00" level=info msg="Storage schema is already up to date"
time="2022-03-09T23:29:41-05:00" level=info msg="Listening for non-TLS connections on '0.0.0.0:9091' paths '/' and '/authelia'"
panic: runtime error: invalid memory address or nil pointer dereference

[signal SIGSEGV: segmentation violation code=0x1 addr=0x200 pc=0x557412a947a7]

goroutine 838 [running]:
github.com/valyala/fasthttp.(*RequestCtx).Done(0x557412644568)
github.com/valyala/fasthttp@v1.33.0/server.go:2691 +0x7
context.propagateCancel.func1()
context/context.go:280 +0x50
created by context.propagateCancel
context/context.go:278 +0x1d0
time="2022-03-09T23:37:03-05:00" level=info msg="Authelia v4.34.3 is starting"
time="2022-03-09T23:37:03-05:00" level=info msg="Log severity set to info"
time="2022-03-09T23:37:03-05:00" level=info msg="Storage schema is being checked for updates"
time="2022-03-09T23:37:03-05:00" level=info msg="Storage schema is already up to date"
time="2022-03-09T23:37:03-05:00" level=info msg="Authelia v4.34.3 is starting"
time="2022-03-09T23:37:03-05:00" level=info msg="Log severity set to info"
time="2022-03-09T23:37:03-05:00" level=info msg="Storage schema is being checked for updates"
time="2022-03-09T23:37:03-05:00" level=info msg="Storage schema is already up to date"
time="2022-03-09T23:37:04-05:00" level=info msg="Listening for non-TLS connections on '0.0.0.0:9091' paths '/' and '/authelia'"
panic: runtime error: invalid memory address or nil pointer dereference

[signal SIGSEGV: segmentation violation code=0x1 addr=0x200 pc=0x5608d842d7a7]

goroutine 969 [running]:
github.com/valyala/fasthttp.(*RequestCtx).Done(0x5608d7fdd568)
github.com/valyala/fasthttp@v1.33.0/server.go:2691 +0x7
context.propagateCancel.func1()
context/context.go:280 +0x50
created by context.propagateCancel
context/context.go:278 +0x1d0
time="2022-03-09T23:43:53-05:00" level=info msg="Authelia v4.34.3 is starting"
time="2022-03-09T23:43:53-05:00" level=info msg="Log severity set to info"
time="2022-03-09T23:43:53-05:00" level=info msg="Storage schema is being checked for updates"
time="2022-03-09T23:43:53-05:00" level=info msg="Storage schema is already up to date"
time="2022-03-09T23:43:53-05:00" level=info msg="Authelia v4.34.3 is starting"
time="2022-03-09T23:43:53-05:00" level=info msg="Log severity set to info"
time="2022-03-09T23:43:53-05:00" level=info msg="Storage schema is being checked for updates"
time="2022-03-09T23:43:53-05:00" level=info msg="Storage schema is already up to date"
time="2022-03-09T23:43:54-05:00" level=info msg="Listening for non-TLS connections on '0.0.0.0:9091' paths '/' and '/authelia'"
panic: runtime error: invalid memory address or nil pointer dereference

[signal SIGSEGV: segmentation violation code=0x1 addr=0x200 pc=0x555d960f07a7]

goroutine 507 [running]:
github.com/valyala/fasthttp.(*RequestCtx).Done(0x555d95ca0568)
github.com/valyala/fasthttp@v1.33.0/server.go:2691 +0x7
context.propagateCancel.func1()
context/context.go:280 +0x50
created by context.propagateCancel
context/context.go:278 +0x1d0
time="2022-03-09T23:45:25-05:00" level=info msg="Authelia v4.34.3 is starting"
time="2022-03-09T23:45:25-05:00" level=info msg="Log severity set to info"
time="2022-03-09T23:45:25-05:00" level=info msg="Storage schema is being checked for updates"
time="2022-03-09T23:45:25-05:00" level=info msg="Storage schema is already up to date"
time="2022-03-09T23:45:26-05:00" level=info msg="Listening for non-TLS connections on '0.0.0.0:9091' paths '/' and '/authelia'"
panic: runtime error: invalid memory address or nil pointer dereference

[signal SIGSEGV: segmentation violation code=0x1 addr=0x200 pc=0x5607915b27a7]

goroutine 1841 [running]:
github.com/valyala/fasthttp.(*RequestCtx).Done(0x560791162568)
github.com/valyala/fasthttp@v1.33.0/server.go:2691 +0x7
context.propagateCancel.func1()
context/context.go:280 +0x50
created by context.propagateCancel
context/context.go:278 +0x1d0
```

Okay, so now it's inconsistent for when it works and when it doesn't. I had it work two times in a row before failing like before with `There was an issue retrieving global configuration` then after restarting the container it worked about a dozen times before throwing up a new red warning. `Failed to initiate security key sign in process`

Restarted the container and about 2 dozen logout logins later and it followed the same pattern. global configuration first and then a failed to initiate.

**Comment by james-d-elliott** [comment]
Can you confirm it's fixed with v4.34.4? A dependency was seemingly causing this and it was bumped to a fixed version in that release.

**Comment by yourjelly** [comment]
I just saw the update come in on my server will test in the morning.

**Comment by yourjelly** [comment]
I've not gotten either of the previous issues on v4.34.4
However, now I am getting security key not recognized depending on which device I'm connecting with. I registered the key to anthelia on my work computer and it was not recognized when I tried to log in with it on my Chromebook at home. I reregistered the key on my Chromebook and was able to use it to log in, but then the key was not recognized by anthelia on my work computer.
